docs/content/documentation/use.md

```diff
@@ -440,6 +440,9 @@ If we quote them, then we end up sending ""colname"" to the backend instead of "
 Fully qualified class name of the class implementing the AuthenticationPlugin interface. If this is null, the password 
 value in the connection properties will be used.
 
+* **`useWithTimezone (`*boolean*`)`** *Default `false`*\
+Map time with timezone and timestamp with timezone to `TIME_WITH_TIMEZONE` and `TIMESTAMP_WITH_TIMEZONE`
```

Comment:
I am afraid these naming and the description do not clarify the meaning of the parameter to the end users. It does not clarify the change changes backward compatibility either.

A bit of a problem is that the parameter impacts both `TIME_WITH_TIMEZONE` and `TIMESTAMP_WITH_TIMEZONE`, so it is hard to select a good name.

WDYT of `mapTimestamptzAndTimeTzTypesToJavaSqlWithTimeZone=true|false`?

An alternative option could be `metadata.jdbctype.timetz=TIME | TIME_WITH_TIMEZONE`, `metadata.jdbctype.timestamptz=TIMESTAMP | TIMESTAMP_WITH_TIMEZONE`.



I suggest using `enum` parameter like `mapTimeStampTz

Reply:
I prefer the second option. I'm not a fan of long names.